test/jdk/java/util/Calendar/SupplementalJapaneseEraTest.java

```diff
@@ -67,6 +67,22 @@ public static void main(String[] args) {
     }
 
     private static void testProperty() {
+        // JDK-8350646: Ensure that IAE is thrown for out of range era, when
+        // additional era is defined
+        try {
+            new Calendar.Builder()
+                    .setCalendarType("japanese")
+                    .setFields(ERA, 7)
```

Comment:
Sorry I suggested this immediate value "7", but this can be `JapaneseEra.values().length + 2` which might be more meaningful

Reply:
Agree that is better; it's pushed.